packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -3089,4 +3089,43 @@ export class API extends EventEmitter {
       );
     });
   }
+
+  changellyGetPairsParams(data): Promise<any> {
```

Comment:
not on this PR, but we should separate integrations with core functions.

core.ts
integrations/changelly.ts
integrations/other.ts

Reply:
also in BWS's server.ts.